### DISCUSSION TITLE
Integrate with Resend official package

### DISCUSSION THREAD

**Discussion by excessivecoding** [epicweb-dev/epic-stack#385] [General]
Resend now support a [official Node package](https://github.com/resendlabs/resend-node).

right now the Epic Stack uses `fetch` to the API which i think is fine for the simple use case but integrating with the official package could be better for future improvement to come?

wdyt?

**Comment by kentcdodds** [comment]
I prefer using the rest API because it's more straightforward to mock and the SDK doesn't really offer any reasonable benefit.

Thanks though!

**Reply by excessivecoding** [reply]
np, completely understand. for future ref, the mocking is still working with the official SDK as it's using the same API endpoint under the hood.

**Reply by kentcdodds** [reply]
Yeah, I'm aware. It's still more of a black box for no benefit 🤷‍♂️
